stream-chat-android-ui-components/src/main/res/values/attrs_channel_list_view.xml

```diff
@@ -2,6 +2,7 @@
 <resources>
     <!--ChannelListView-->
     <declare-styleable name="ChannelListView">
+        <attr name="streamUiChannelsItemSeparatorDrawable" format="reference" />
         <attr name="streamUiChannelTitleTextSize" format="dimension|reference" />
         <attr name="streamUiLastMessageDateTextSize" format="dimension|reference" />
         <attr name="streamUiLastMessageTextSize" format="dimension|reference" />
```

Comment:
These attributes are on `SimpleChannelListView` instead of on `ChannelListView`, so I think they are currently not settable by client code

Reply:
I've double-checked that and the attributes are working as expected. That's because we are creating `SimpleChannelListView` in code and its passing attributes to `SimpleChannelListView`

Reply:
Hmm, wow. And it receives the attributes that are set on its parent view?

Reply:
yes, https://github.com/GetStream/stream-chat-android/pull/1363/files#diff-e798047d8cb772e63ff90bf9d89e0e54266be3fd50e9a9326f270b38d83d6676R37 those attrs comes from the parent (ChannelListView)